microsoft-edge/devtools-guide-chromium/remote-debugging/windows.md

```diff
@@ -43,66 +39,68 @@ Install the [Remote Tools for Microsoft Edge (Beta)](https://www.microsoft.com/s
 
-Note the machine IP address and connection port displayed under **Connect using:**.  The IP address in the image below is `192.168.86.78` and the connection port is `50080`.
+Note the machine IP address and connection port displayed under **Connect using:**.  The IP address in the image below is `192.168.86.78` and the connection port is `50080`:
 
-:::image type="complex" source="../media/remote-debugging-windows-media-host-settings-ip-address.msft.png" alt-text="Note the IP address and connection port in the Settings" lightbox="../media/remote-debugging-windows-media-host-settings-ip-address.msft.png":::
-   Note the IP address and connection port in the **Settings**
-:::image-end:::
+:::image type="content" source="../media/remote-debugging-windows-media-host-settings-ip-address.msft.png" alt-text="Note the IP address and connection port in the Settings." lightbox="../media/remote-debugging-windows-media-host-settings-ip-address.msft.png":::
 
 You enter the information on the client (debugger) device in the [next section](#step-2-set-up-the-client-debugger-machine).  Open tabs in Microsoft Edge and [Progressive Web Apps (PWAs)](../progressive-web-apps/index.md) on the host (debuggee) machine that you want to debug from the client (debugger) machine.
 
 
 <!-- ====================================================================== -->
 ## Step 2: Set up the client (debugger machine)
 
-The client or debugger machine is the device you want to debug from.  This device may be your daily development machine or it may just be your PC or MacBook when working from home.
+The client or debugger machine is the device you want to debug from.  This device may be your daily development machine, or it may just be your PC or MacBook when working from home.
+
+1.  To set up the client (debugger) machine, install Microsoft Edge from [this page](https://www.microsoft.com/edge) if you haven't already.
 
-To set up the client (debugger) machine, install Microsoft Edge from [this page](https://www.microsoft.com/edge) if you have not already.  If you are using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
+1.  If you're using a pre-installed version of Microsoft Edge on the host (debuggee) machine, verify that you have Microsoft Edge (Chromium) and not Microsoft Edge (EdgeHTML).  A quick way to check is to load `edge://settings/help` in the browser and confirm that the version number is 75 or higher.
 
-Now navigate to `edge://flags` in Microsoft Edge.  In **Search flags**, type in **Enable remote Windows device debugging in edge://inspect**.  Set that flag to **Enabled**.  Then, choose the **Restart** button to restart Microsoft Edge.
```

Comment:
@bwalderman - I removed this step (in Step 2 section), and removed the screenshot.  This is the motivation for this PR.

Reply:
LGTM